File: src/main/java/top/zbeboy/isy/web/bean/data/student/StudentBean.java
package top.zbeboy.isy.web.bean.data.student;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.Student;

import java.sql.Date;
import java.sql.Timestamp;

/**
 * Created by lenovo on 2016-09-27.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class StudentBean extends Student {
    private Integer schoolId;
    private String schoolName;
    private Byte schoolIsDel;

    private Integer collegeId;
    private String collegeName;
    private String collegeAddress;
    private Byte collegeIsDel;

    private Integer departmentId;
    private String departmentName;
    private Byte departmentIsDel;

    private Integer scienceId;
    private String scienceName;
    private Byte scienceIsDel;

    private String organizeName;
    private Byte organizeIsDel;
    private String grade;

    private String password;
    private Byte enabled;
    private Integer usersTypeId;
    private String realName;
    private String mobile;
    private String avatar;
    private Byte verifyMailbox;
    private String mailboxVerifyCode;
    private String passwordResetKey;
    private Timestamp mailboxVerifyValid;
    private Timestamp passwordResetKeyValid;
    private String langKey;
    private Date joinDate;

    private String nationName;

    private String politicalLandscapeName;

    private String roleName;

    // 楼号
    private String ridgepole;
    // 宿舍号
    private String dorm;
}

File: src/main/java/top/zbeboy/isy/web/bean/data/student/StudentBean.kt
package top.zbeboy.isy.web.bean.data.student

import top.zbeboy.isy.domain.tables.pojos.Student
import java.sql.Date
import java.sql.Timestamp

/**
 * Created by zbeboy 2017-11-19 .
 **/
class StudentBean : Student() {
    var schoolId: Int? = null
    var schoolName: String? = null
    var schoolIsDel: Byte? = null

    var collegeId: Int? = null
    var collegeName: String? = null
    var collegeAddress: String? = null
    var collegeIsDel: Byte? = null

    var departmentId: Int? = null
    var departmentName: String? = null
    var departmentIsDel: Byte? = null

    var scienceId: Int? = null
    var scienceName: String? = null
    var scienceIsDel: Byte? = null

    var organizeName: String? = null
    var organizeIsDel: Byte? = null
    var grade: String? = null

    var password: String? = null
    var enabled: Byte? = null
    var usersTypeId: Int? = null
    var realName: String? = null
    var mobile: String? = null
    var avatar: String? = null
    var verifyMailbox: Byte? = null
    var mailboxVerifyCode: String? = null
    var passwordResetKey: String? = null
    var mailboxVerifyValid: Timestamp? = null
    var passwordResetKeyValid: Timestamp? = null
    var langKey: String? = null
    var joinDate: Date? = null

    var nationName: String? = null

    var politicalLandscapeName: String? = null

    var roleName: String? = null

    var idCard: String? = null

    // 楼号
    var ridgepole: String? = null
    // 宿舍号
    var dorm: String? = null
}